sympy/integrals/transforms.py

```diff
@@ -1885,15 +1899,34 @@ def laplace_transform(f, t, s, legacy_matrix=True, **hints):
     Examples
     ========
 
-    >>> from sympy import DiracDelta, exp, laplace_transform
-    >>> from sympy.abc import t, s, a
+    >>> from sympy import DiracDelta, exp, laplace_transform, diff, Function, symbols
+    >>> y = Function('y')
+    >>> Y = Function('Y')
+    >>> z = Function("z")
+    >>> Z = Function("Z")
+    >>> s, t, a, b, c, d, z0, z1, y0 = symbols('s, t, a, b, c, d, z0, z1, y0')
```

Comment:
Most of these aren't needed until lower down. I suggest to define them where they are needed so that we can keep the first examples simple.

Also multiple functions can be created like:
```python
y, Y, z, Z = symbols('y, Y, z, Z', cls=Function)
```

Reply:
Thanks! I did not know this yet.